src/Admin/BaseFieldDescription.php

```diff
@@ -416,25 +418,75 @@ public function getFieldValue($object, $fieldName)
             return $this->getFieldValue($child, substr($fieldName, $dotPos + 1));
         }
 
-        $getters = [];
-        $parameters = [];
-
         // prefer method name given in the code option
         if ($this->getOption('code')) {
-            $getters[] = $this->getOption('code');
+            $getter = $this->getOption('code');
+
+            if (!method_exists($object, $getter)) {
+                @trigger_error(
+                    'Passing a non-existing method in the "code" option is deprecated'
```

Comment:
IMO, we could accept also accessible property names too.

Reply:
Why would you use the propertyName directly then ?

Reply:
What I mean is, we could allow `ShowMapper::add('bar')` to be resolved in the model even if the getter, isser or hasser don't exist, but the `Foo::$bar` property is accessible. I think this case is specially useful for PHP +7.4, where the properties can directly declare its type and there is no need of getters or setters for such purpose.

Reply:
It's already the case to me.

add('bar') looks for getter, isser, hasser or property.
See https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Admin/BaseFieldDescription.php#L457

And property accessor works the same way.

`code` is an option to pass a custom getter name, so it should be a method.

Reply:
Oh, I didn't realize that, sorry. Just dismiss my comment then. Thank you!